python/lsst/obs/base/_instrument.py

```diff
@@ -398,6 +398,24 @@ def makeDataIdTranslatorFactory(self) -> TranslatorFactory:
         """
         raise NotImplementedError("Must be implemented by derived classes.")
 
+    @classmethod
+    def constructCollectionName(cls, label: str) -> str:
+        """Get the instrument-specific collection string to use as derived
+        from the supplied label.
+
+        Parameters
+        ----------
+        label : `str`
+            String to be combined with the instrument name to form a
+            collection name.
+
+        Returns
+        -------
+        name : `str`
+            Collection name to use that includes the instrument name.
+        """
+        return f"{cls.getName()}/{label}"
+
```

Comment:
This is fine, but I found it mildly amusing to note that this wouldn't have actually abstracted over the old convention, which would have needed `{label1}/{cls.getName()}/{label2}`.

Reply:
All the cases I am replacing were label1/instrument: calib and raw.